lib/elixir/test/elixir/kernel/quote_test.exs

```diff
@@ -263,11 +263,6 @@ defmodule Kernel.QuoteTest do
     assert quote(do: &1.foo) == quote(do: &1.foo)
   end
 
-  test "operators slash arity" do
```

Comment:
Is it OK to simply remove the test?
We could just run `make format` to fix the build (with the new tokenizer it became `quote(do: &+/2)`), if it makes any sense to keep a test for this oc. I have no opinion, just asking for confirmation :slightly_smiling_face:

Reply:
Yes, it is ok. We do cover `& &/1` and `& +/2` elsewhere. That was always the syntax!

Reply:
Noted, thanks for clarifying! :heart: